cli/pkg/local/server.go

```diff
@@ -513,6 +514,34 @@ func (s *Server) Deploy(ctx context.Context, r *connect.Request[localv1.DeployRe
 	}), nil
 }
 
+func (s *Server) GetCurrentUser(ctx context.Context, r *connect.Request[localv1.GetCurrentUserRequest]) (*connect.Response[localv1.GetCurrentUserResponse], error) {
+	if !s.app.ch.IsAuthenticated() {
+		return nil, errors.New("not authenticated as a user")
+	}
```

Comment:
Can it return `nil` for the current user for consistency with the admin RPC implementation? (The admin implementation returns `nil` since not being logged is a normal case, not an error)